code/modules/modular_computers/file_system/programs/command/comm.dm

```diff
@@ -112,6 +113,20 @@
 		return program.can_run(user, program.computer.get_network())
 	return 1
 
+/datum/nano_module/program/comm/proc/get_shunt()
+	var/obj/comp = program.computer.get_physical_host()
+	var/obj/effect/overmap/visitable/ship/sector = map_sectors["[comp.z]"]
```

Comment:
Would it be worth nullchecking `comp` and `program.computer`? Not sure how robust modular programs are.

Reply:
I'm not really certain myself.